tests/functional/test_libraries.py

```diff
@@ -87,6 +87,7 @@ def test_gevent(pyi_builder):
         """)
 
 
+@xfail(is_py36, reason='Fails on python 3.6')
```

Comment:
I'm not that convinced about with skipping these. This will make us ignore this problem. Or doe these tests fail due to gevent resp. pylint not supporting Python 3.6 yet?